src/components/VersionNumber.js

```diff
@@ -9,7 +9,12 @@ import { Wrapper } from './Wrapper.js';
 export const VersionNumber = () => {
   return (
     <Wrapper>
-      <RegularText small style={styles.version}>
+      <RegularText
+        small
+        style={styles.version}
+        accessible={true}
```

Comment:
**good to know**:
for passing a prop with the value `true`, it can be shortend with just `accessible`, without explicitly giving the value true.
we already do that in many places in the code.

`<RegularText accessible>` is the same as `<RegularText accessible={true}>`